src/components/ListChampions.jsx

```diff
@@ -0,0 +1,34 @@
+import { useState, useEffect } from 'react';
+
+function List() {
+  const [champions, setChampions] = useState([]);
+
+  useEffect(() => {
+    // Récupération de la liste de tous les champions depuis l'API Data Dragon
+    fetch('http://ddragon.leagueoflegends.com/cdn/13.6.1/data/en_US/champion.json')
+      .then(response => response.json())
+      .then(data => {
+        // Stockage de la liste des noms de champions dont le nom commence par "A"
+        const filteredChampions = Object.values(data.data)
```

Comment:
```suggestion
        const filteredChampions = data.filter( // ...
```
Pas besoin du `Object.values`